### PR TITLE
Remove full signature of member in description

### PR DESCRIPTION
**Pull request by vmoyseenko**
#### Purpose

Remove signature from description for `ZeroTouch` libraries.
untitled1

#### Additional references

[MAGN-6317](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6317) Remove full signature of member in description

#### Reviewers

@Benglin, please, take a look.

### CODE REVIEW

File: src/DynamoCore/Library/FunctionDescriptor.cs [hunk 1]
```diff
@@ -170,7 +170,7 @@ public string Summary
         /// </summary>
         public string Description
         {
-            get { return !String.IsNullOrEmpty(Summary) ? Summary + "\n\n" + Signature : Signature; }
+            get { return !String.IsNullOrEmpty(Summary) ? Summary : string.Empty; }
```
Comment (Benglin) [inline src/DynamoCore/Library/FunctionDescriptor.cs]: I would prefer `get { return String.IsNullOrEmpty(Summary) ? string.Empty : Summary; }`, but this is okay too.

**Comment by Benglin** [comment]
Thanks @vmoyseenko, LGTM!

**Comment by dronovdmitry** [comment]
Thanks @vmoyseenko . How can i find you on FB? 
I am dynamo scripterr and architector:
https://www.facebook.com/dronov.d.u
https://www.youtube.com/channel/UCg5ny7d27XbjMUHMuEdOtQA

**Comment by vmoyseenko** [comment]
Hi @dronovdmitry, here is my account in FB https://www.facebook.com/vmoyseyenko. Let's have a talk.

**Pull request merged**